components/event-block/EventBlockSpeakers.vue

```diff
@@ -0,0 +1,268 @@
+<template>
+  <div>
+    <img
+      v-if="showWaveMarker"
+      src="/event-title-wave.svg"
+      width="111"
+      height="35"
+      alt=""
+      class="event-block-speakers__icon"
+    >
+
+    <h3
+      class="event-block-speakers__title"
+      :class="{
+        'event-block-speakers__title--orange': titleColor === 'orange',
+        'event-block-speakers__title--blue': titleColor === 'blue',
+      }"
+    >
+      {{ title }}
+    </h3>
+
+    <p class="event-block-speakers__subtitle">
+      {{ subtitle }}
+    </p>
+
+    <swiper
+      :options="swiperOptions"
+      class="event-block-speakers__carousel"
+    >
+      <swiper-slide
+        v-for="speaker in speakers"
+        :key="speaker.id"
+        class="event-block-speakers__speaker"
+      >
+        <div class="event-block-speakers__speaker-wrapper">
+          <div class="event-block-speakers__speaker-heading">
+            <img
+              :src="`${speaker.image.url}?auto=compress&w=95&h=95&fit=crop&crop=entropy`"
+              alt=""
+              class="event-block-speakers__speaker-image"
+            >
+
+            <div class="event-block-speakers__speaker-meta">
+              <h4 class="event-block-speakers__speaker-name">
+                {{ speaker.name }}
+              </h4>
+
+              <p class="event-block-speakers__speaker-organization">
+                {{ speaker.organization }}
+              </p>
+            </div>
+          </div>
+
+          <h5 class="event-block-speakers__speaker-label">
+            {{ speaker.subjectLabel }}
+          </h5>
+
+          <p class="event-block-speakers__speaker-subject">
+            {{ speaker.subject }}
+          </p>
+        </div>
+      </swiper-slide>
+
+      <div
+        slot="button-prev"
+        class="swiper-button-prev"
+      />
+
+      <div
+        slot="button-next"
+        class="swiper-button-next"
+      />
+
+      <div
+        slot="scrollbar"
+        class="swiper-scrollbar"
+      />
+    </swiper>
+  </div>
+</template>
+
+<script>
+  import 'swiper/css/swiper.css';
```

Comment:
Should this also be done in `EventBlockChapters`?

Reply:
This import is also done in `EventBlockChapters`.